common/smoke-tests/smoke-test.yml

```diff
@@ -80,6 +80,10 @@ jobs:
 
 
     steps:
+      - template: /eng/pipelines/templates/steps/update-version-fix.yml
+        parameters:
```

Comment:
To reduce duplication and the number of lines we have to touch I suggest defaulting the VMAgent parameter to `$(OSVmImage)` in the template so we don't have to pass it in explicitly in all our usages.